modules/smtp/site.js

```diff
@@ -137,8 +137,6 @@ var save_compose_state = function(no_files, notice) {
         return;
     }
 
-    $('.smtp_send_placeholder').prop('disabled', true);
-    $('.smtp_send_placeholder').addClass('disabled_input');
```

Comment:
If you remove this feature, you should also take care of the "undisabling" code below...

Another thing to check - is it OK to submit the form while draft is saved as ajax request? What happens to the draft? I suppose it will be left on the IMAP server as compose_draft_id will be still empty when submitting the compose form. In that case, isn't it better to submit another ajax request before submitting the form - only if the previous one hasn't completed and make the draft be deleted?